source/blog/2018-07-16-ember-3-3-released.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Ember 3.3 Released
+author: Melanie Sumner, Brendan McLoughlin
+tags: Releases, 2018, 3, 3.3
+responsive: true
+---
+
+Today the Ember project is releasing version 3.3.0 of Ember.js, Ember Data, and Ember CLI. We know, 3.2 seems like it just came out yesterday! But we wanted to (stay? get back?) on track. So 3.3.0 is here! 
+
+This release kicks off the 3.4 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.3
+
+Ember.js 3.3 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There are one (1) new feature, three (3) deprecations, and eight (8) bugfixes in this version.
+
+#### New Features (1)
+
+**Implement optional jQuery Integration (1 of 1)**
+
+See [emberjs/rfcs#294](https://github.com/emberjs/rfcs/blob/master/text/0294-optional-jquery.md) to get more prose
+
+#### Deprecations (3)
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+**Operation "make jQuery optional" continues (1 of 3)**
+We implemented a deprecation that removes access from `jQuery.Event#originalEvent`
+Read the full RFC: [emberjs/rfcs#294](https://github.com/emberjs/rfcs/blob/master/text/0294-optional-jquery.md).
+
+**deprecation of three private, unused classes (2 of 3)**
+Implemented deprecation of `Ember.Map`, `Ember.MapWithDefault`, and `Ember.OrderedSet` because not only were they private classes, they had also remained unused in Ember for some time now. Read the full (rendered) RFC: [emberjs/rfc#237](https://github.com/Serabe/rfcs/blob/dbc64fd66c602c68cfe03e91bfa2d031f4907423/text/0000-deprecation-ember-map.md)
```

Comment:
Link to merged RFC https://github.com/emberjs/rfcs/blob/master/text/0237-deprecation-ember-map.md